src/rockstar.js

```diff
@@ -82,14 +83,40 @@ function groupBlocks(statements) {
 	return ret
 }
 
+function computeDependencies(statements) {
+	const addReadLine = statements.some(s => s.t === 'Listen');
+	const fs = addReadLine;
+	
+	return [
+		fs && `const $fs = require('fs');`,
+		addReadLine && `
+		function readLineSync() {
```

Comment:
This function should probably be `$readLineSync()` the way `$fs` is, to prevent clashes with names used by the program. (Not that I expect someone to name their Rockstar function `readLineSync` but just in case.)